app/models/slot_availability.rb

```diff
@@ -3,30 +3,10 @@ class SlotAvailability
 
   def initialize(prison:, use_nomis_slots: false)
     @prison = prison
```

Comment:
I'm 👍 on the refactor.

However I've noticed that we are using `prison` as an instance variable and as a parameter, would be good to make it consistent.